README.md

```diff
@@ -57,6 +57,28 @@ Click _Edit config_. This creates a config file called `claude_desktop_config.js
 }
 ```
 
+To use remote MCP server:
+```json
+{
+  "mcpServers": {
+    "bitrise": {
+      "command": "npx",
+      "args": [
+        "mcp-remote",
+        "https://localhost:8000/",
+        "--header",
+        "Authorization: <YOUR_TOKEN>",
+        "--header",
+        "x-bitrise-enabled-api-groups: cache-items,pipelines"
+      ],
+      "env": {
+        "PATH": "/.../..../nodejs/20.13.1/bin:/bin"
```

Comment:
This won't be the same for everyone, right? In that case I would remove it and make a comment that you might need to pass an absolute path to `command` or set the `PATH` `env` var accordingly.

Reply:
I actually documented `uvx` like `<absolute path to>/uvx` bit it was removed for some reason.

Reply:
Also, we should probably document how one can run the MCP server in HTTP mode as well in this section to paint the full picture.